common/build/eslint-config-fluid/package.json

```diff
@@ -15,8 +15,6 @@
 		"build": "npm run print-config",
 		"cleanup-printed-configs": "node ./scripts/cleanup-printed-configs.js ./printed-configs",
 		"format": "npm run prettier:fix",
-		"lint": "npm run prettier && npm run eslint",
-		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
```

Comment:
Nit: just call it out in the description. If I got it right, they're unnecessary because the package doesn't have actual source code (just config files)?

Reply:
Correct. I'll include this in the commit comment.